### PR TITLE
[MIG] web_widget_url_advanced: Migration to 17.0

### PR DESCRIPTION
**Pull request by nedaszilinskas**

### CODE REVIEW

**Comment by nedaszilinskas** [comment]
@simahawk @alexey-pelykh @Peerapong-Supasompob I'm looking for help reviewing this PR. Would you have some time? 🙏

**Comment by simahawk** [comment]
@ivantodorovich could you please have a look?

File: web_widget_url_advanced/static/src/fields/url_field.esm.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+patch(UrlField.props, {
+    text_field: {type: String, optional: true},
+    prefix_name: {type: String, optional: true},
+});
+
+patch(UrlField.prototype, {
+    _get_text_field() {
+        if (this.props.text_field) {
+            let field_value = this.props.record.data[this.props.text_field];
+            if (typeof field_value === "object" && field_value.length == 2) {
```
Comment (ivantodorovich) [inline web_widget_url_advanced/static/src/fields/url_field.esm.js:18]:
```suggestion
            if (Array.isArray(field_value) && field_value.length == 2) {
```

seems to narrow the conditions

**Review by ivantodorovich — APPROVED** [review]
technically lg, thanks!

File: web_widget_url_advanced/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "version": "17.0.1.0.0",
+    "license": "LGPL-3",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/web",
+    "depends": ["web"],
+    "data": [],
+    "assets": {
+        "web.assets_backend": [
+            "web_widget_url_advanced/static/src/fields/url_field.esm.js",
+            "web_widget_url_advanced/static/src/fields/url_field.xml",
```
Comment (vvrossem) [inline web_widget_url_advanced/__manifest__.py:17]:
```suggestion
            "web_widget_url_advanced/static/src/fields/**/*",
```

**Review by vvrossem — COMMENTED** [review]
small suggestion: update both `static/description/[before|after].png` screenshots with 17.0 ones

**Comment by vvrossem** [comment]
Thank you for migrating this module.
Can you please ensure that pre-commit auto fixes from https://github.com/OCA/web/pull/2837/commits/7ef81261bafed41cf18e8dab8ec714a07ccd9748 are squashed into https://github.com/OCA/web/pull/2837/commits/16682427af32b9ffde6520ab6d4c9c547f477014 ?

**Comment by yankinmax** [comment]
@nedaszilinskas Can you pls apply suggestions?

**Review by yankinmax — APPROVED** [review]

**Comment by yankinmax** [comment]
@nedaszilinskas @simahawk @vvrossem @ivantodorovich 
Your suggestions are applied.
This PR will be superseded in the purpose to fast-forward by:
- [ ] https://github.com/OCA/web/pull/2918

**Pull request closed without merging**